Formula/bup.rb

```diff
@@ -36,16 +36,33 @@ class Bup < Formula
     sha256 "0fe2d45ba43b00a41cd73f8be321a44936dc1aba233dee979f17a042b83eb6dc"
   end
 
+  if OS.linux?
+    resource "pyxattr" do
+      url "https://files.pythonhosted.org/packages/cf/b1/7ed931d98b5a91a59b69fcc2860e5b720a22ed1ddb85268415181c9b0986/pyxattr-0.7.1.tar.gz"
+      sha256 "965388dd629334e850aa989a67d2360ec8257cfe8f67d07c29f980d3152f2882"
+    end
+  end
+
   def install
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    python_config = Formula["python@3.8"].opt_libexec/"bin/python-config"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python3.8/site-packages"
+    ENV.append "CFLAGS", " #{`#{python_config} --cflags`.chomp}"
+    ENV.append "LDFLAGS", " #{`#{python_config} --ldflags`.chomp}"
+    ENV["PYTHON"] = "python3"
```

Comment:
This should probably be this, since otherwise it could match system python
```suggestion
    ENV["PYTHON"] = Formula["python@3.8"].opt_bin/"python3"
```

Reply:
With `depends_on "python@3.8"` doesn't that introduce our `python` in the `PATH` before the system python?